lib/segment/src/data_types/primitive.rs

```diff
@@ -25,12 +26,12 @@ pub trait PrimitiveVectorElement:
     fn datatype() -> VectorStorageDatatype;
 
     fn from_float_multivector(
-        multivector: Cow<TypedMultiDenseVector<VectorElementType>>,
-    ) -> Cow<TypedMultiDenseVector<Self>>;
+        multivector: CowMultiVector<VectorElementType>,
```

Comment:
All this file is about `Cow` -> `CowMultiVector`. Reason why we remove `Cow` here is described above